submissions/victorAienobe/php/hobbyArrays.php

```diff
@@ -0,0 +1,11 @@
+<?php
+
+    $names = ['Pele','Maradona','Messi','Ronaldo','Neymar','Hazard','Debruyne','Aguero'];
+
+    $hobbies = ['movies','music','chatting','video games','swimming','skating','football','basketball'];
+
+    foreach(array_combine($names,$hobbies) as $key => $value) {
```

Comment:
Well done. Kindly format this block of code

Reply:
alright, It's been formatted